core/src/ledger/parameters/storage.rs

```diff
@@ -119,6 +120,14 @@ pub fn is_max_proposal_bytes_key(key: &Key) -> bool {
     is_max_proposal_bytes_key_at_addr(key, &ADDRESS)
 }
 
+/// Returns if the key is the max signature per transacton key
+pub fn is_max_signatures_per_transaction_key(key: &Key) -> bool {
+    matches!(&key.segments[..], [
+        DbKeySeg::AddressSeg(addr),
+        DbKeySeg::StringSeg(max_signatures_per_transaction),
+    ] if addr == &ADDRESS && max_signatures_per_transaction == Keys::VALUES.max_signatures_per_transaction)
```

Comment:
```suggestion
    is_max_signatures_per_transaction_key_at_addr(key, &ADDRESS)
```
generated from `derive(StorageKeys)`